pages/search.js

```diff
@@ -85,19 +84,23 @@ export default function Search({ data: { users, tags, randUsers } }) {
       const nameLower = user.name.toLowerCase();
       const usernameLower = user.username.toLowerCase();
       const userTagsString = user.tags.join(", ").toLowerCase();
-      
+      const userLocationString = user.location ? [user.location.provided]
```

Comment:
There is no reason to create an array then perform a join on it.
This combination essentially does nothing, because the array only has one element.

Reply:
While there is a possibility the location information will change in the future, there is no benefit to adding additional complexity now that will still need to be modified in the future.

Reply:
Sure! I have fixed it.